src/App.js

```diff
@@ -6,24 +6,34 @@ import TopNav from "./components/TopNav";
 import ImageSlider from "./components/ImageSlider";
 import ProductDetail from "./components/ProductDetail";
 import Footer from "./components/Footer";
-import changeCategory from "./index";
 
 function App(props) {
-	var currentCategory = props.currentCategory;
+	const products = props.products;
+	var productsDiv = '';
+
+	productsDiv = products.map((product, index) => {
```

Comment:
for future reference. do not check in code that causes an error and breaks the web page